oraibridge/chain.json

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "oraibridge",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://orai.io/",
+  "pretty_name": "OraiBridge",
+  "chain_id": "oraibridge-subnet-2",
+  "bech32_prefix": "oraib",
+  "daemon_name": "gravity",
+  "node_home": "$HOME/.gravity",
+  "key_algos": ["secp256k1"],
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "uoraib",
+        "fixed_min_gas_price": 0,
+        "low_gas_price": 0,
+        "average_gas_price": 0,
+        "high_gas_price": 0
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "uoraib"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/oraichain/Gravity-Bridge",
+    "recommended_version": "v1.0.2.2",
+    "compatible_versions": ["v1.0.2.2"],
+    "binaries": {
+      "linux/arm64": "https://github.com/oraichain/Gravity-Bridge/releases/download/v1.0.2.2/gravity_arm64?checksum=sha256:14ca829756f3e9cf9bccc24ea3aa394bcb029532ee7b4d34f9719613c2ed265a"
+    },
+    "genesis": {
+      "genesis_url": "https://raw.githubusercontent.com/oraichain/oraichain-static-files/master/mainnet-static-files/genesis.json"
+    },
+    "versions": [
+      {
+        "name": "v1.0.2.2",
+        "height": 9381254,
+        "recommended_version": "v1.0.2.2",
+        "compatible_versions": ["v1.0.2.2"],
+        "binaries": {
+          "linux/arm64": "https://github.com/oraichain/Gravity-Bridge/releases/download/v1.0.2.2/gravity_arm64?checksum=sha256:14ca829756f3e9cf9bccc24ea3aa394bcb029532ee7b4d34f9719613c2ed265a"
+        }
+      }
+    ]
+  },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai.svg"
+  },
+  "peers": {
+    "seeds": [
+      {
+        "id": "b0873bb0e2f25fd7cca0a54c7db76ea5adee2abb",
+        "address": "18.221.105.194:26656",
+        "provider": "oraichain-team"
+      }
+    ],
+    "persistent_peers": [
+      {
+        "id": "b0873bb0e2f25fd7cca0a54c7db76ea5adee2abb",
+        "address": "18.221.105.194:26656",
+        "provider": "oraichain-team"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://bridge.rpc.orai.io",
+        "provider": "oraichain-team"
+      }
+    ],
+    "rest": [
+      {
+        "address": "https://bridge.lcd.orai.io",
+        "provider": "oraichain-team"
+      }
+    ],
+    "grpc": [
+      {
+        "address": "http://18.221.105.194:9090",
+        "provider": "oraichain-team"
+      }
+    ]
+  },
+  "explorers": [
+    {
+      "kind": "oraiscan",
+      "url": "https://scan.bridge.orai.io",
+      "tx_page": "https://scan.bridge.orai.io/txs/${txHash}"
+    }
+  ],
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/orai.png",
```

Comment:
Since we're looking to the /oraichain/images/ directory for the orai logo, I don't see a need to copy them to /oraibridge/images/ directory. Can we remove those please?